File: Assignment14/Question2.c
# include<stdio.h>

void Pattern(int iRow , int iCol)
{
    int i = 0;
    int iNum = 1;
    for(i = 0 ; i < iRow ; i++)
    {   
        if (i % 2 == 0) 
        {
            iNum = 2; 
        } 
        else 
        {
            iNum = 1; 
        }
        {
            int j = 0;
            for(j = 0 ; j < iCol ; j++)
            {
                printf("%d\t",iNum);
                iNum = iNum + 2 ; 
            }    
        }
    printf("\n");
    }
}
int main()
{
    int iValue1 = 0;
    int iValue2 = 0;

    printf("Enter the number of rows : ");
    scanf("%d",&iValue1);

    printf("Enter the number of Coloumns : ");
    scanf("%d",&iValue2);

    Pattern(iValue1,iValue2);
    
    return 0;    
}

File: Assignment47/Question2.java
import java.util.*;

class Pattern 
{
    public void Pattern(int iRow , int iCol)
    {
        int i = 0 , j = 0;

        for(i = 1 ; i <= iRow ; i++)
        {
            for(j = 1 ; j <= iCol; j++)
            {
                System.out.print(j+"\t");
            }
            System.out.println();
        }

    }
}

class Question2
{
    public static void main(String A[])
    {
        int iValue1 = 0 , iValue2 = 0 ;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter The Number Of Rows :");
        iValue1 = sobj.nextInt();

        
        System.out.println("Enter The Number Of Coloumns :");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Pattern(iValue1,iValue2);
    }
}